internal/persistence/sql/migrations/single_table_test.go

```diff
@@ -226,7 +226,7 @@ func TestToSingleTableMigrator_HasLegacyTable(t *testing.T) {
 
 				ln, err := m.LegacyNamespaces(ctx)
 				require.NoError(t, err)
-				assert.Equal(t, nspaces, ln)
+				assert.ElementsMatch(t, nspaces, ln)
```

Comment:
Is this migration affected by your changes? It is meant to be removed on a future release anyways, so we can also do it now. Just needs some documentation on how to migrate then (i.e. first use Keto v0.7.0 and continue from there).

Reply:
It was a flaky test, since the order of the two slices was not guaranteed.

Reply:
Ah I see :+1: